src/utils/displayFunctions.tsx

```diff
@@ -8,9 +8,35 @@ import { IST_ICON } from 'assets/assets';
 import type { BrandInfo } from 'store/app';
 import { wellKnownBrands } from 'config';
 
-export const notifyError = err => {
+import { Id as ToastId, toast } from 'react-toastify';
+import { DeliverTxResponse } from '@cosmjs/stargate';
+
+export const notifySigning = () =>
+  toast.loading(<p>Awaiting sign and broadcast…</p>);
+
+export const notifySuccess = (toastId: ToastId, tx: DeliverTxResponse) => {
+  const txUrl = `https://bigdipper.live/agoric/transactions/${tx.transactionHash}`;
+  toast.update(toastId, {
```

Comment:
We should make the success and error toasts dismissible

Reply:
agreed